js/bybit.js

```diff
@@ -48,7 +48,7 @@ module.exports = class bybit extends Exchange {
                 'fetchDepositAddressesByNetwork': true,
```

Comment:
@sc0Vu 
Can't we also update fetchMarkets with this method? https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_instrhead

⬆️  just bear in mind that I don't think that v3 and unified margin are not necessarily the same thing, meaning that I think we can use v3 to do non-unified-margin things (the opposite is not true), like for instance create a swap order using the normal margin mode

So I think we should probably check one more option, `version`, and apply the following logic:
- if version === v3 or unifiedMargin 
  - use v3 endpoint to load markets, trades, tickers, etc 
  -  if unifiedMargin -> use v3 unified margin endpoints to order related methods
  - if not unified margin -> use v3 "regular" endpoints to order related methods

Reply:
I'm not sure whether the trades/tickers are the same between v3 and v2 endpoints. Probably bybit will upgrade the v2 apis to v3 (eg, v2 linear to v3 linear)? If the price is different, would that break `createOrder` functionality?

If they are the same, we can also check the version. BTW, should we compare v3 in other public methods?

Reply:
After check https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_instrhead, it seems we can only fetch one category market at the time. It might need 3 requests to fetch linear/inverse/options markets.

Reply:
> I'm not sure whether the trades/tickers are the same between v3 and v2 endpoints. Probably bybit will upgrade the v2 apis to v3 (eg, v2 linear to v3 linear)? If the price is different, would that break `createOrder` functionality?
> 
> If they are the same, we can also check the version. BTW, should we compare v3 in other public methods?

If they are not the same we have to update them accordingly. Basically, if the user selects v3 version we have to use everything from v3 (tickers, precisions, etc, etc)